test/bugs/gh-1116.js

```diff
@@ -0,0 +1,128 @@
+// https://github.com/swagger-api/swagger-js/issues/1116
+import { buildRequest } from '../../src/execute';
+
+describe('test accept header for oas 3', () => {
```

Comment:
Can you please incorporate these OAS 3 tests into https://github.com/swagger-api/swagger-js/tree/master/test/oas3/execute directory by creating new file `build-request.js`?